cypress/integration/full-run.spec.js

```diff
@@ -112,21 +112,24 @@ describe('should perform functions on the review page', () => {
         cy.get('select[name="TimeSlot"]').select(timeString)
       })
       cy.get(nextButton).should('be.enabled').and('be.visible')
+      cy.wait(2000)
       cy.get(nextButton).click()
-      cy.url().should('contains', '/review')
+      cy.wait(2000)
```

Comment:
All the commented out code makes this a bit hard to read, but it looks like you have a `wait(2000)` call at the very end of the file. What does it do?

Reply:
the wait is to make sure the server has responded before the test continues, as it seems that the test keeps going regardless and then there is an error because the drop-downs are empty.